samples/tab-personal-sso-quickstart/csharp_dotnetcore/Views/Shared/_Layout.cshtml

```diff
@@ -11,10 +11,11 @@
             @RenderBody()
         </main>
     </div>
+    <script src="https://res.cdn.office.net/teams-js/2.0.0-beta.6/js/MicrosoftTeams.min.js"
```

Comment:
Lets update to GA beta CDN location also aiming to have it by EOW.

Reply:
Yes changed to latest GA 2.0 version